README.md

```diff
@@ -88,11 +87,11 @@ The tests require [Zephyr](https://github.com/zephyrproject-rtos/zephyr) (if you
 Build system
 ------------
 
-When calling cddl-gen with the argument `--output-cmake <file path>`, a cmake file will be created at that location.
+When calling ZCBOR with the argument `--output-cmake <file path>`, a cmake file will be created at that location.
```

Comment:
Should we have the name of the executable upper case? Here you can say that the executable is not what is being targeted, but the project, but I would say that using the name of the executable would be better since one might think that the command you are executing is `ZCBOR`